src/components/Timeline/Publication/Actions/Collect/ReferralAlert.tsx

```diff
@@ -0,0 +1,28 @@
+import Slug from '@components/Shared/Slug';
```

Comment:
**question:** can we reuse this from `Publication` folder?

Reply:
Done